examples/pymc3_howto/lasso_block_update.ipynb

```diff
@@ -20,6 +20,10 @@
    "outputs": [],
```

Comment:
<p>I would use <code>az.plot_pair</code>  here, it can also use hexbin mode, some examples: https://arviz-devs.github.io/arviz/examples/plot_pair_hex.html and https://arviz-devs.github.io/arviz/examples/plot_joint.html. Side advantages, I think with that we can use <code>return_inferencedata=True</code>  directly and the plot will be automatically labeled</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>